src/components/Wikipage.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import Logo from "./Logo";
+import Searchbar from "./Searchbar";
+import { searchWiki } from "../api";
```

Comment:
unused import, remove it.

Reply:
also should make sure across the app that no unused import is there. eslint can help you there as well.